morse_challenge.rb

```diff
@@ -0,0 +1,9 @@
+def decode(string)
+  morse_code = { '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E', '..-.' => 'F', '--.' => 'G',
+                 '....' => 'H', '..' => 'I', '.---' => 'J', '-.-' => 'K', '.-..' => 'L', '--' => 'M',
+                 '-.' => 'N', '---' => 'O', '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R', '...' => 'S',
+                 '-' => 'T', '..-' => 'U', '...-' => 'V', '.--' => 'W', '-..-' => 'X', '-.--' => 'Y', '--..' => 'Z' }
+  string.split.map { |letter| morse_code[letter] }.join(' ')
+end
+
+puts decode('-- -.--   -. .- -- .')
```

Comment:
- [ ] Great job working on this milestone. As per project requirements, you were supposed to add individual methods for:
- #decode_char method which translates Morse code symbols into letters.
**Example**: `p decode_char('.-')` or `decode_char('.-')` should return `"A"`
- #decode_word method which combines translated chars into a single word to be returned as the final output. It should reuse the `#decode_char` method
**Example**: `p decode_word('-- -.--')` or `decode_word('-- -.--')` should return `"MY"`.

At the end add the following lines of code below and run `ruby morse_challenge.rb` from the terminal
```
p decode_char('.-') or decode_char('.-')
p decode_word('-- -.--') or decode_word('-- -.--')
p decode("-- -.--   -. .- -- .") or decode("-- -.--   -. .- -- .")
p decode(".-   -... --- -..-   ..-. ..- .-.. .-..    --- ..-.    .-. ..- -... .. . ...") or
decode(".-   -... --- -..-   ..-. ..- .-.. .-..    --- ..-.    .-. ..- -... .. . ...")
```

You should have the results below:
```
"A"
"MY"
"MY NAME"
"A BOX FULL OF RUBIES"
```